modules/apps/oauth2-provider/oauth2-provider-web/src/main/resources/content/Language_fr.properties

```diff
@@ -29,7 +29,7 @@ authorize-oauth2-application=Autoriser une application OAuth 2
 authorize-x=Autoriser {0}
 can-be-assigned-via-a-global-scope=La portée de cette ressource peut être affectée via une portée globale.
 changed=Modifié
-choose-one-of-the-following-global-scopes-that-include-this-resource-scope=Choisissez l'un des périmètres globaux suivants qui incluent la portée de cette ressource :
+choose-one-of-the-following-global-scopes-that-include-this-resource-scope=Choisissez l'un des périmètres globaux suivants qui incluent la portée de cette ressource :
```

Comment:
Unsure if `0xa0` (non-breaking space) is critical here, to avoid breaking lines.